### ISSUE TITLE
Improve `Select` component rendering when with more than 10,000 options

### ISSUE THREAD

**Issue by signup2k** [causalens/dara#278]
```
Select(
    value=select_var,
    # items=[Item(label=f'Choice {i}', value=f'{i}') for i in range(1, 10000)],
    items=[f'Choice {i}' for i in range(1, 10000)],
    multiselect=True,
    searchable=True
)
```

When use the `Select` component with a very large number of items (say, more than 10000 as I tested), the component seems very sluggish.

Whether using the items with or without `Item()` have the very same sluggish effect.

my case of using large number of options in select:
a select for user to select stock, etf, idx symbols.

**Comment by krzysztof-causalens** [comment]
Hello,

Thanks for reporting the issue. It's one we're aware of and are actively working to fix. 
In fact, a PR for multiselect has already been merged on our UI side https://github.com/causalens/dara-ui/pull/98 and should be available in latest Dara v1.8.3+ (`@darajs/components` v1.9.1). Let us know if the new release feels better!
Similar performance improvements will be implemented for other `Select` types soon, we should have a release with these within a week or so.

Thanks for checking out Dara, let us know if you have any feedback :)

**Comment by signup2k** [comment]
Thanks for ur quickly reply. Glad to know this issue will be solved with the next release. I am gonna close this issue.

Dara is awesome framework.
